chromadb/api/models/CollectionCommon.py

```diff
@@ -441,35 +399,28 @@ def _process_add_request(
         )
 
         normalized_embeddings = (
-            self._normalize_embeddings(unpacked_embedding_set["embeddings"])
-            if unpacked_embedding_set["embeddings"] is not None
+            self._normalize_embeddings(unpacked_record_set["embeddings"])
+            if unpacked_record_set["embeddings"] is not None
             else None
         )
 
-        self._validate_embedding_set(
-            ids=unpacked_embedding_set["ids"],
-            embeddings=normalized_embeddings,
-            metadatas=unpacked_embedding_set["metadatas"],
-            documents=unpacked_embedding_set["documents"],
-            images=unpacked_embedding_set["images"],
-            uris=unpacked_embedding_set["uris"],
+        unpacked_record_set["embeddings"] = normalized_embeddings
+
+        self._validate_record_set(
```

Comment:
Let's make the same logic changes as `_process_upsert_or_update_request`

Reply:
it is already there